sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -388,93 +404,57 @@ architecture.
 |-
 |NVIDIA Hopper architecture.
 
-|`amd_gpu_gfx700`
+|`amd_gpu_gfx700` to `amd_gpu_gfx702`
 |-
-|AMD GCN GFX7 (Sea Islands (CI)) architecture.
+|AMD GCN 2.0 architecture.
 
-|`amd_gpu_gfx701`
+|`amd_gpu_gfx801`, `amd_gpu_gfx802`, `amd_gpu_gfx805`, `amd_gpu_gfx810`
 |-
-|AMD GCN GFX7 (Sea Islands (CI)) architecture.
+|AMD GCN 3.0 architecture.
 
-|`amd_gpu_gfx702`
+`amd_gpu_gfx803`
```

Comment:
The table formatting is wrong from this point forward.

Reply:
Thank you for your reviews!